README.md

```diff
@@ -212,7 +212,70 @@ dnationcloud/dnation-kubernetes-monitoring-stack \
 -f multicluster-config/observer-values.yaml \
 -f <custom-observer-values-sample.yaml>
 ```
+# Openshift support
+## Tested versions
+||dNation monitoring v1.3|dNation monitoring v1.4|dNation monitoring v2.3|
+|-|-|-|-|
+|Openshift v4.7|||✓|
+## Installation
+To install the chart on an openshift cluster, use additional [values for openshift](/chart/values-openshift.yaml)
+```shell
+ helm upgrade --install -n dnation-monitoring dnation-monitoring chart -f chart/values-openshift.yaml
```

Comment:
I think here you should provide installation  from helm repository like we have it in our README. Same applies for multicluster setup.